.travis.yml

```diff
@@ -1,13 +1,26 @@
 language: dart
 
-dart:
-  - 1.24.3
-  - stable
-
 # Re-use downloaded pub packages everywhere.
 cache:
   directories:
     - $HOME/.pub-cache
 
-script:
-  - ./validate.sh
+jobs:
```

Comment:
Do we prefer to use current pattern where we specify task per version or rather run the same task for all version?

Reply:
The reason I did this is because the formatting can differ based on the Dart SDK version. So with this setup, we're only running the format step on stable. I also made it so that dependency_validator only runs on stable since its behavior shouldn't differ based on the SDK version at all so running it once is sufficient